templates/registration/logout.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Logout</h1>
+  <form method="post" action="{% url 'taxi:logout' %}">
```

Comment:
The URL name `taxi:logout` may not be correct if you are using Django's built-in authentication URLs. Typically, the logout URL is `logout` without the app namespace. Ensure that the URL name matches the one defined in your URL configuration.